packages/internal/bridge/sdk/src/constants/bridges.ts

```diff
@@ -2,6 +2,11 @@
 
 import { AxelarChainDetails, BridgeInstance } from '../types';
 
+/**
+ * @constant {string} NATIVE - The native token representation.
+ */
+export const NATIVE = 'NATIVE';
```

Comment:
the use of this string constant feels like it will be error-prone. It is declared elsewhere as well (in lowercase). have we considered abstracting native tokens into a type instead?

Reply:
Good question. If we look at how the DEX SDK does it, [they have a specific type for the native token](https://github.com/immutable/ts-immutable-sdk/blob/main/packages/internal/dex/sdk/src/types/index.ts#L126-L137), which is then used in `Coin`, which is of type `ERC20 | Native`.

They still have to, at some point, [deal with some sort of stringy way of representing](https://github.com/immutable/ts-immutable-sdk/blob/main/packages/internal/dex/sdk/src/exchange.ts#L145-L156) that we want to use the native asset, though. Not sure if it's avoidable entirely.

Reply:
Yes, the intention is to avoid any breaking change. Checkout [uses the string of 'native'](https://github.com/immutable/ts-immutable-sdk/blob/e573fcc6c926e85f29f8160be3273fff6fd58580/packages/checkout/widgets-lib/src/lib/constants.ts#L7)  across different components so this I think ideally this should be done in a separate PR to avoid any potential issues of this change.

Reply:
Yeah we can still accept the string, but then convert it into an appropriate type early on in our SDK's flow